Exercise_6/tests/test.js

```diff
@@ -0,0 +1,123 @@
+const assert = require('assert');
+const chai = require('chai');
+const calculate = require('../ReversePolishNotation').calculate;
+
+describe('ReversePolishNotationTests',()=> {
+
+    describe('Sums', ()=>{
+
+        it('should return 5',()=> {
+            assert.equal(calculate("3 2 +"), 5);
+        });
+
+        it('should return 10',()=>{
+            assert.equal(calculate("-30 10 20 + 10 + +"),10);
+        });
+
+        it('should return -6', ()=>{
+            assert.equal(calculate("-1 -2 -3 + +"),-6);
+        });
+
+        it('should return 0', ()=>{
+            assert.equal(calculate("0"),0);
+        });
+
+        it('should return 1000', ()=>{
+            assert.equal(calculate("300 200 500 + +"),1000);
+        })
+    });
+
+    describe('Subtractions', ()=>{
+
+        it('should return 3',()=> {
+            assert.equal(calculate("7 4 -"), 3);
+        });
+
+        it('should return -20',()=>{
+            assert.equal(calculate("10 20 - 10 -"),-20);
+        });
+
+        it('should return -2', ()=>{
+            assert.equal(calculate("-1 -2 -3 - -"),-2);
+        });
+
+        it('should return 0', ()=>{
+            assert.equal(calculate("5 5 -"),0);
+        });
+
+        it('should return 600', ()=>{
+            assert.equal(calculate("300 200 500 - -"),600);
+        })
+    });
+
+
+    describe('Multiplications', ()=>{
+
+        it('should return 142',()=> {
+            assert.equal(calculate("71 2 *"), 142);
+        });
+
+        it('should return -300',()=>{
+            assert.equal(calculate("10 30 * -1 *"),-300);
+        });
+
+        it('should return -15', ()=>{
+            assert.equal(calculate("-5 -3 -1 * *"),-15);
+        });
+
+        it('should return 0', ()=>{
+            assert.equal(calculate("5 0 *"),0);
+        });
+
+        it('should return 30000000', ()=>{
+            assert.equal(calculate("300 200 500 * *"),30000000);
+        })
+    });
+
+    describe('Mixed operators', ()=>{
```

Comment:
I think one test would be enough. Just make some really long line with all operators.